src/services/enterpriseService.ts

```
@@ -0,0 +1,312 @@
+
+    await prisma.bulkTransferJob.update({
+      where: { id: job.id },
+      data: {
+        processedRows,
+        successCount,
+        failureCount,
+      },
+    });
+  }
+
+  const failureReport = allResults.filter((result) => result.status === "failure");
```

ESLint

[error] Replace `(result)·=>·result.status·===·"failure"` with `⏎····(result)·=>·result.status·===·"failure",⏎··`

(prettier/prettier)